### PR TITLE
Skip flaky accelerate test

### PR DESCRIPTION
**Pull request by amyeroberts**
# What does this PR do?

Skipping another flaky accelerate tests until values have been aligned with accelerate library 

cc @muellerzr

### CODE REVIEW

File: examples/pytorch/test_pytorch_examples.py [hunk 1]
```diff
@@ -200,6 +201,7 @@ def test_run_mlm(self):
             result = get_results(tmp_dir)
             self.assertLess(result["perplexity"], 42)
 
+    @unittest.skip("This test is flaky on CI.")
```
Comment (ydshieh) [inline examples/pytorch/test_pytorch_examples.py:204]: Having something like  `# TODO: fixme (zack)` would be great, but if we expect this to be fixed quickly, it's fine.

**Review by ydshieh — APPROVED** [review]
Thank you!

**Comment by muellerzr** [comment]
@amyeroberts can you link to a failure? I thought #27378 fixed it :( (I tried to run every combo to see what that value would be)

**Comment by amyeroberts** [comment]
@muellerzr Sure! Here's two CI runs with the test failing after rebasing on main: 
* https://app.circleci.com/pipelines/github/huggingface/transformers/77801/workflows/57ee7a82-f95d-43e2-8431-44420be74491/jobs/991432
* https://app.circleci.com/pipelines/github/huggingface/transformers/77800/workflows/147b815d-30df-41c9-90ad-5013a78e0e1c/jobs/991478

File: examples/pytorch/test_pytorch_examples.py (hunk 1 above)
Comment (amyeroberts) [inline examples/pytorch/test_pytorch_examples.py:204]:
```suggestion
    @unittest.skip("TODO: FIXME (Zack) - This test is flaky on CI.")
```

Comment (amyeroberts) [reply examples/pytorch/test_pytorch_examples.py]: Agreed - done!

**Comment by muellerzr** [comment]
Ah pytorch test, got it. I'll see if I can recreate this and see during the alignment PR what that looks like

**Comment by muellerzr** [comment]
I can recreate locally it failing with 0.51 (same as the accelerate example that fixed), I can't recreate this 0.67 result :(

**Comment by amyeroberts** [comment]
> I can recreate locally it failing with 0.51 (same as the accelerate example that fixed), I can't recreate this 0.67 result :(

@ydshieh In this case - what would you recommend? Hold off until we can replicate and know we've found all the issues?

**Review by ArthurZucker — COMMENTED** [review]
The failing test for me is this one [examples.pytorch.test_pytorch_examples.ExamplesTests ](https://app.circleci.com/pipelines/github/huggingface/transformers/77817/workflows/9b991a8f-6fa4-45f3-bc00-c2f83da15195/jobs/991680/tests). Let's just skip this and let @muellerzr work on the fix!

**Comment by muellerzr** [comment]
re; slack: this is not the right issue. tl;dr the way we install the git pip dependencies does not actually grab the latest main commit, a different fix is needed.

**Comment by muellerzr** [comment]
https://github.com/huggingface/transformers/pull/27404 will solve this in full :)

**Pull request closed without merging**